### ISSUE TITLE
Failing jest tests with added @babel/runtime dependency in 3.2.1

### ISSUE THREAD

**Issue by denyo** [mui-org/material-ui-pickers#1219]
## Environment

| Tech                 | Version |
| -------------------- | ------- |
| @material-ui/pickers | 3.2.1        |
| material-ui          | 4.3.0        |
| React                | 16.8.6        |
| Browser              | node@10.12.0 / jest        |
| Peer library         | date-io/moment@1.3.6, moment@2.24.0        |  |
| typescript | 3.5.2 |

## Steps to reproduce

Upgrade from 3.2.0 to 3.2.1.

## Expected behavior

Jest tests are passing like before.

## Actual behavior

All Jest tests are failing

```
 FAIL  src/App.test.tsx
  ● Test suite failed to run

    Jest encountered an unexpected token

    This usually means that you are trying to import a file which Jest cannot parse, e.g. it's not plain JavaScript.

    By default, if Jest sees a Babel config, it will use that to transform your files, ignoring "node_modules".

    Here's what you can do:
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/en/configuration.html

    Details:

    ~/some_path/node_modules/@babel/runtime/helpers/esm/extends.js:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){export default function _extends() {
                                                                                             ^^^^^^

    SyntaxError: Unexpected token export

      at ScriptTransformer._transformAndBuildScript (node_modules/jest-runtime/build/script_transformer.js:403:17)
      at Object.<anonymous> (node_modules/@material-ui/pickers/dist/material-ui-pickers.js:7:32)
```

It appears that this is caused by the new dependency `"@babel/runtime": "^7.2.0",` which made it into the bundled `package.json`.

## Live example

None.

**Comment by y3sh** [comment]
This is also affecting upstream libs, e.g. if you import material-table, which imports material-ui-pickers, project tests may fail. This doesn't happen on macos/linux, but it is happening on windows.

Looking forward to release so I can notify maintainers.

See last line of stacktrace:
```
export default function _extends() {
^^^^^^

SyntaxError: Unexpected token export
    at Module._compile (internal/modules/cjs/loader.js:721:23)
    at Module._compile (.......\node_modules\pirates\lib\index.js:99:24)
    at Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Object.newLoader [as .js] (.....\node_modules\pirates\lib\index.js:104:7)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (....\node_modules\@material-ui\pickers\dist\material-ui-pickers.js:7:32)
```

**Comment by jamesmillerburgess** [comment]
This version of also breaks every application using the meteor build tool as reported here: https://github.com/meteor/meteor/issues/10655

We are currently upgrading from v2, which did not have this issue.

**Comment by dmtrKovalenko** [comment]
Released in v3.2.2
